src/index.ts

```diff
@@ -19,6 +19,7 @@ import React from 'react';
  * - `['a', 1, 1000]`
  * - `['a', 2]`
  */
+
```

Comment:
this line should be deleted, because it detaches the documentation above from the code being documented. This would break typedoc output.

Reply:
okay will be updating in sometime, thanks

Reply:
Updated.